python/subscribe/subscribe.py

```diff
@@ -22,13 +22,29 @@ def on_signals(signals_in_frame: SignalsInFrame):
     client.on_signals = on_signals
 
     try:
-        subscription = client.subscribe(signal_names=signals, namespaces=namespaces, changed_values_only=False)
+
+        def to_signal_id(signal: str):
```

Comment:
Should we remove this logic from here and let the work in https://github.com/remotivelabs/remotivelabs-apis/pull/35/files handle this?

Reply:
The idea was to be backwards compatible since it might be used in other places. I.e I have some stuff in scania that relies on this. Not sure...